openwisp_controller/config/admin.py

```diff
@@ -156,7 +156,7 @@ class TemplateAdmin(MultitenantAdminMixin, AbstractTemplateAdmin):
 
 TemplateAdmin.list_display.insert(1, 'organization')
 TemplateAdmin.list_filter.insert(0, ('organization', MultitenantOrgFilter))
-TemplateAdmin.fields.insert(1, 'organization')
+TemplateAdmin.fieldsets[0][1]['fields'].insert(1, 'organization')
```

Comment:
Ah, I see the change. It will be obsolete very soon when https://github.com/openwisp/openwisp-controller/pull/190 is merged.

Reply:
I thought so as I saw #190 now. I think than this can be safely closed as #190 seems to be merged soon enough :smile:.